README.md

```diff
@@ -146,11 +144,9 @@ curl base_url/community/ \
 
 **Request :**
 ```
-curl base_url/user/award/ \
+curl base_url/award/<sample_user_name> \
```

Comment:
`award` should be plural -> `awards`.

Is this the endpoint for getting awards for a specific user in a specific community? If so, then my comments is the as above @roymoran @omkarjc27